docs/setup.md

```diff
@@ -0,0 +1,104 @@
+# Set-up Service
+
+## **Prerequisites**
+
+### Python and Flask version
+See requirements.in file
+
+## **Clone the repository**
+
+### Create a Virtual environment
+
+    python3 -m venv .venv
+
+### Enter the virtual environment
+
+...either macOS using bash:
+
+    source .venv/bin/activate
+
+...or if on Windows using Command Prompt:
+
+    .venv\Scripts\activate.bat
+
+## **Dependencies**
+
+### Install dependencies
+
+From the top-level directory enter the following command to install pip and the dependencies of the project, including local development dependencies:
+
+    python3 -m pip install --upgrade pip && pip install -r requirements-dev.txt
+
+### Add dependencies
+
+requirements-dev.txt and requirements.txt are updated using [pip-tools pip-compile](https://github.com/jazzband/pip-tools).
+
+To update project requirements add the dependencies into the .in files (not the .txt files) and run (in the following order):
+
+    pip-compile requirements.in
+
+    pip-compile requirements-dev.in
+
+If the packages listed in the .txt files do not meet those specified in the .in files (including version restrictions), the previous commands will update the .txt files accordingly.
+
+Then to install the upgraded dependencies:
+
+    pip install -r requirements-dev.txt
+
+### Upgrade existing dependencies
+
+To ensure that dependencies are using the most recent compatible versions available we can run a pip-tools upgrade. To upgrade dependency versions in the existing .txt files, run (in the following order):
```

Comment:
Should we be suggesting using dependabot to manage this for us?

Reply:
good point, dependabot should certainly be our primary tool. I will add that thanks